.travis.yml

```diff
@@ -1,14 +1,10 @@
----
-os: linux
-dist: bionic
-
-jobs:
-  include:
-    - language: python
-      python: 3.8
-      cache: pip
-      install: "pip install -r requirements.txt"
-      script: sh run_tests.sh
+language: python
```

Comment:
As @jwflory worked on modernizing the `.travis.yml` file for the CI, Please add: 

```
os: linux
dist: bionic

jobs:
  include:
     - language: python
       ...
```

Reply:
There. Done. :)